rac_aspace/serializers.py

```diff
@@ -1,60 +1,61 @@
 import argparse
-from configparser import ConfigParser
-# import csv
-import os
+import csv
 
 
-config = ConfigParser()
-config.read("local_settings.cfg")
+class Serializer:
 
-file_path = os.path.join(
-    os.path.abspath(os.path.dirname(__file__)), config.get(
-        "Destinations", "filename")
-)
+    def __init__(self, filename, delimiter, data):
```

Comment:
```suggestion
    def __init__(self, filename, filemode="w"):
```
I'd consider passing data into the method currently called `run()`. This would allow you to instantiate a writer once and then write multiple times in a script. so, for example:
```
writer = CSVWriter('path/to/file.csv')
data = get_stuff_from_as()
writer.run(data)
more_data = get_more_stuff()
writer.run(more_data)
```

I'm also suggesting an optional filemode argument so that you can append to an existing file if desired.